pubspec.yaml

```diff
@@ -13,7 +13,7 @@ dependencies:
   path: ^1.8.0
   source_maps: ^0.10.10
   stack_trace: ^1.10.0
-  vm_service: '>=11.9.0 <13.0.0'
+  vm_service: '>=12.0.0 <13.0.0'
```

Comment:
Nit: If we're constraining vm_service to a single major revision, we should just use `^12.0.0` instead of an explicit range.